test/integration/short/load-balancing-tests.js

```diff
@@ -13,18 +13,20 @@ var WhiteListPolicy = loadBalancing.WhiteListPolicy;
 
 context('with a reusable 3 node cluster', function () {
   this.timeout(180000);
-  helper.setup(3, {
-    queries: [
+  var setupQueries = [
       'CREATE KEYSPACE ks_simple_rp1 WITH replication = {\'class\': \'SimpleStrategy\', \'replication_factor\' : 1}',
       'CREATE KEYSPACE ks_network_rp1 WITH replication = {\'class\': \'NetworkTopologyStrategy\', \'datacenter1\' : 1}',
       'CREATE KEYSPACE ks_network_rp2 WITH replication = {\'class\': \'NetworkTopologyStrategy\', \'datacenter1\' : 2}',
       'CREATE TABLE ks_simple_rp1.table_a (id int primary key, name int)',
       'CREATE TABLE ks_network_rp1.table_b (id int primary key, name int)',
       'CREATE TABLE ks_network_rp2.table_c (id int primary key, name int)',
+    ];
+  if (helper.isCassandraGreaterThan('2.0')) {
```

Comment:
I think that given that C* 1.2 is no longer a supported version, we can filter this test fixture out (ie: `vdescribe()`).